tools/keycloak/sync/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Pims.Tools.Keycloak.Sync": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "workingDirectory": "C:\\git\\PSP\\tools\\keycloak\\sync"
```

Comment:
is `workingDirectory` required? this will not match the local for each developer